docs/reference/commandline/run.md

```diff
@@ -244,6 +244,23 @@ $ docker run -it --pid=container:my-redis my_strace_docker_image bash
 $ strace -p 1
 ```
 
+### <a name="uts"></a> UTS settings (--uts)
+
+```text
+--uts=""  : Set the UTS namespace mode for the container
+            'host': use the host's UTS namespace inside the container
+```
+
+The UTS namespace is for setting the hostname and the domain that's visible to
+running processes in that namespace. By default, all containers, including
+those with `--network=host`, have their own UTS namespace. The `host` setting
+will result in the container using the same UTS namespace as the host. Note
```

Comment:
We should avoid future tense.

Reply:
carried in https://github.com/docker/cli/pull/4712